integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -96,6 +97,9 @@ public final class AlluxioFuseUtils {
   public static final long MODE_NOT_SET_VALUE = -1;
   public static final String MACFUSE_SUPPORT_MINIMUM_OS_VERSION = "10.9";
 
+  private static final Pattern QUOTA_EXCEEDED_PATTERN =
+      Pattern.compile("(.*)The DiskSpace quota of(.*)exceeded([\\s\\S]*)");
```

Comment:
Is it true others would return the same error message? Or there is a way to check according to a return value other than a string match?

Reply:
The pattern is only applicable to ozone and hdfs. Unable to identify the exception classes of ozone and hdfs, so check the message.

Reply:
could it be converted to a standard type in the UFS? The message matching is not a good way.

Reply:
@Haoning-Sun can you help give us the whole workflow of where this exception is thrown originally and how it's reach to FUSE in the end, and why this exception is thrown (when what happens). This help us figure out whether they are better ways to pass the original exception (may be a better exception that refers to quota exceed) to FUSE and directly translate this exception to error code

Reply:
- Create bucket and volume in ozone and use 'setquota' to set space quota for volume.
- Mount the ozone location in the alluxio.
- Start a fuse process which its write type is CACHE_THROUGH  to mount the alluxio path to a local directory.
- Copy a file exceeding the quota to the mount directory. The following exception will be thrown and return EIO.
```
2023-01-05 22:45:53,236 ERROR AlluxioFuseUtils - Failed to Fuse.Write(path=/test.txt,fd=1,size=65536,offset=11468800) with unexpected throwable:
java.lang.RuntimeException: alluxio.exception.status.UnknownException: The DiskSpace quota of bucket:testexceeded: quotaInBytes: 104857600 Bytes but diskspace consumed: 805306368 Bytes. (GrpcDataWriter{request=type: UFS_FILE
...
```